stf/tasks/create_basecase_forecast.py

```diff
@@ -25,7 +25,7 @@
 
 def main():
     with TaskContext(__file__) as context:
-        model_type = "xgb"
+        model_type = "lgb"
 
```

Comment:
Should remain xgb, maybe add lgb like this: model_type = ["xgb", "lgb"]

Reply:
I'll change it, thanks!